tests/Datasets/LabeledTest.php

```diff
@@ -83,7 +83,8 @@ public function build() : void
      */
     public function fromIterator() : void
     {
-        $dataset = Labeled::fromIterator(new NDJSON('tests/test.ndjson'));
+        $dir = realpath(dirname(dirname(__DIR__)));
```

Comment:
Hmm, this looks a little messy and repetitive. I'll push a cleaner solution in a minute